File: servico_basico/src/java/modelo/Sexo.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package modelo;

/**
 *
 * @author superalunocmc
 */
public class Sexo extends Objeto {
    
    private String descricao;

    public Sexo() {
    }

    public Sexo(String descricao, int id) {
        super(id);
        this.descricao = descricao;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }
}

File: Cadastro de Pessoa/Cadastro de Pessoa/Modelo/Sexo.cs
namespace ClienteREST.Modelo
{
    public class Sexo:Objeto
    {
        public string descricao { get; set; }

        public Sexo() { }

        public Sexo(int id, string descricao)
        {
            this.id = id;
            this.descricao = descricao;
        }
    }
}
